scipy/optimize/tests/test__shgo.py

```diff
@@ -749,7 +749,7 @@ def test_15_min_every_iter(self):
         run_test(test1_1, n=1, iters=7, options=options,
                  sampling_method='sobol')
 
-    def test_16_disp_bounds_minimizer(self):
+    def test_16_disp_bounds_minimizer(self, capsys):
```

Comment:
I wonder whether these `capsys` / `capfd` parameters are worth a code comment - it's not immediate to me that these are built-in `pytest` stuff.

Reply:
I considered it but decided against it, since it's used in a number of places. If you've never seen a fixture it's confusing, but that is equally true for other pytest fixtures. I think you just need to know the rule that if a test function has an argument that doesn't come from a `parametrize` decorator, it's always a fixture.